frontend/pages/event-recaps/[event_recap_id].js

```diff
@@ -0,0 +1,215 @@
+import React, { useState, useEffect } from "react";
+import PageHeader from "../../components/header";
+import styles from "../../styles/event-recap.module.css";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import Accordion from "@material-ui/core/Accordion";
+import AccordionDetails from "@material-ui/core/AccordionDetails";
+import AccordionSummary from "@material-ui/core/AccordionSummary";
+import Typography from "@material-ui/core/Typography";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import AwesomeSlider from "react-awesome-slider";
+import "react-awesome-slider/dist/styles.css";
+import axios from "axios";
+import humps from "humps";
+import Image from "next/image";
+
+const EventRecapPage = (props) => {
+  const [expanded, setExpanded] = useState(false);
+  const [event, setEvent] = useState({});
+  const [loading, setLoading] = useState(true);
+  const [hasResources, setHasResources] = useState(false);
+  const [hasPhotos, setHasPhotos] = useState(false);
+  const [hasFBLink, setHasFBLink] = useState(false);
+  const [imageGalleryFilenames, setImageGalleryFilenames] = useState([]);
+  const [hasEmbeddedVideo, setHasEmbeddedVideo] = useState(false);
+  const [enableGalleryArrows, setEnableGalleryArrows] = useState(true);
+  const [eventNumber, setEventNumber] = useState("0");
+
+  useEffect(() => window.scrollTo(0, 0), []);
+
+  useEffect(() => {
+    setLoading(true);
+    let url = window.location.href.split("/");
+    setEventNumber(url[url.length-1].split("_")[1])
+    let currEventNumber = url[url.length-1].split("_")[1]
+    const fetchPastEvents = async () => {
+      const res = await axios.get("https://wit-database.herokuapp.com/past-events");
+      const allEvents = humps
+        .camelizeKeys(res.data)
+        .filter(
+          // (event) => event.year.toString() === props.match.params.year
+          (event) => event.year.toString() === url[url.length-1].split("_")[0]
```

Comment:
Change url to be split by `/` rather than `-` eg 2022/16 for consistency

Reply:
doneee !!